components/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+export const Footer = () => {
+    const Container = styled.View`
```

Comment:
Styled components should be declared outside of any component (ideally, below declaration of `<Footer />`)